### ISSUE TITLE
Gitprefix v2.1.9 don't work

### ISSUE THREAD

**Issue by gollum23** [syuchan1005/GitPrefix#29]
**Describe the bug**
When I selected a option from dropdown this not change the commit message neither maintain the selected item in dropdown

**Screenshots**
![Screenshot_20200707_181032](https://user-images.githubusercontent.com/1058783/86853995-36bca880-c07d-11ea-9310-bd580a47e5ef.png)
After select any option
![Screenshot_20200707_181111](https://user-images.githubusercontent.com/1058783/86854024-46d48800-c07d-11ea-89ff-43d95ca2df05.png)

**Desktop**
- IDE: Pycharm 2020.1
- JRE: Java 11.0.7+10b765.53 amd64
- JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
- Platform: Manjaro Linux KDE
![Screenshot_20200702_103703](https://user-images.githubusercontent.com/1058783/86379073-fb227880-bc4f-11ea-9fd4-9d842103f671.png)

**.gitprefix file**
```text
:wrench:                Update
:bug:                   Bug Fixing
:horse_racing:          Performance
:construction:          WIP
:warning:               Migration
:lock:                  Security Fix
:recycle:               Refactoring
:art:                   Cosmetic
:books:                 Documentation
:no_entry:              Deletion
:white_check_mark:      Addition
:globe_with_meridians:  Translations
:sparkles:              New Feature
:police_car:            Test
:rocket:                Feature
:arrow_double_up:       Upgrade

```

**Comment by syuchan1005** [comment]
Info stacktrace
```
kotlin.UninitializedPropertyAccessException: lateinit property checkinProjectPanel has not been initialized
	at com.github.syuchan1005.gitprefix.git.commit.GitCommitComboBoxAction$Companion.getCheckinProjectPanel(GitCommitComboBoxAction.kt:64)
	at com.github.syuchan1005.gitprefix.git.commit.GitCommitComboBoxAction$Companion.setCurrent(GitCommitComboBoxAction.kt:68)
	at com.github.syuchan1005.gitprefix.git.commit.GitCommitComboBoxAction$createPopupActionGroup$1.invoke(GitCommitComboBoxAction.kt:42)
	at com.github.syuchan1005.gitprefix.git.commit.GitCommitComboBoxAction$createPopupActionGroup$1.invoke(GitCommitComboBoxAction.kt:13)
	at com.github.syuchan1005.gitprefix.util.PrefixResourceFileUtil$BlockType$Companion$toAction$1.actionPerformed(PrefixResourceFileUtil.kt:169)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:280)
...
```

**Comment by syuchan1005** [comment]
Fixed in v2.1.10.
